src/lib/app.ts

```diff
@@ -193,10 +193,6 @@ export default async function getApp(
     // Setup API routes
     app.use(`${baseUriPath}/`, new IndexRouter(config, services, db).router);
 
-    if (services.openApiService) {
-        services.openApiService.useErrorHandler(app);
-    }
```

Comment:
This can easily be removed because useErrorHandler only handles errors identified as coming from openapi